src/dbnode/storage/bootstrap.go

```diff
@@ -148,6 +154,7 @@ func (m *bootstrapManager) Bootstrap() error {
 	// on its own course so that the load of ticking and flushing is more spread out
 	// across the cluster.
 
+	m.lastBootstrapCompletionTime = m.nowFn()
```

Comment:
Should we only set this if `multiErr.FinalError() == nil`?

Reply:
resolved offline, its ok as is because of how our bootstrapping process works, but we also have an additional sanity check at the clustered DB level that makes sure each shard for each namespace is bootstrapped